src/core/ComplexFilter/index.tsx

```diff
@@ -55,12 +55,13 @@ export default function ComplexFilter<
     getInitialValue()
   );
 
-  const [pendingValue, setPendingValue] = useState<
-    Value<DefaultMenuSelectOption, true>
-  >([]);
+  const [pendingValue, setPendingValue] = useState<DefaultMenuSelectOption[]>(
```

Comment:
change `Value<DefaultMenuSelectOption, true>` to `DefaultMenuSelectOption[]` to be more readable